[Serve] Document WebSocket support in Serve (#36735)
ray-project/ray: doc/source/serve/http-guide.md

## Calling Deployments via HTTP

> When you deploy a Serve application, the [ingress deployment](serve-key-concepts-ingress-deployment) (the one passed to `serve.run`) will be exposed over HTTP.

Reviewer: ```suggestion
When you deploy a Serve application, the [ingress deployment](serve-key-concepts-ingress-deployment) (the one passed to `serve.run`) is exposed over HTTP.
```
